File: Oxygene/Cooper/Android/ActionBar/com.remobjects.ActionBar/SettingsActivity.pas
namespace com.remobjects.actionbar;

interface

uses
  java.util,
  android.app,
  android.content,
  android.os,
  android.util,
  android.view,
  android.widget;

type
  SettingsActivity = public class(Activity)
  private
  protected
  public
    method onCreate(savedInstanceState: Bundle); override;
  end;

implementation

method SettingsActivity.onCreate(savedInstanceState: Bundle);
begin
  inherited;
    //  Create the text view
  var textView: TextView := new TextView(self);
  textView.TextSize := 40;
  textView.Text := "Settings Activity";

  //  Set the text view as the activity layout
  ContentView := textView;

  //Set the Up Button Action in the ActionBar
  getActionBar().setDisplayHomeAsUpEnabled(true);
end;

end.

File: Hydrogene/Cooper/Android/ActionBar/com.remobjects.actionbar/SettingsActivity.cs
using java.util;
using android.app;
using android.content;
using android.os;
using android.util;
using android.view;
using android.widget;

namespace com.remobjects.actionbar
{
	public class SettingsActivity : Activity
	{
		public override void onCreate(Bundle savedInstanceState)
		{
			base.onCreate(savedInstanceState);

			TextView textView = new TextView(this);
			textView.TextSize = 40;
			textView.Text = "Settings Activity";
			ContentView = textView;

			getActionBar().setDisplayHomeAsUpEnabled(true);

		}
	}
}